### PR TITLE
wv-2172 GEDI Remove v1, add v2 AUG 2021 layers

### PR DESCRIPTION
**Pull request by minniewong**
## Description

Fixes WV-2172

Removed Version 1 layers that spanned 2019 APR - 2020 APR: 
GEDI_ISS_L3_Elevation_Mean_Lowest_Mode_201904-202004
GEDI_ISS_L3_Elevation_StdDev_Lowest_Mode_201904-202004
GEDI_ISS_L3_Laser_Footprint_Count_201904-202004
GEDI_ISS_L3_Canopy_Height_Mean_RH100_201904-202004
GEDI_ISS_L3_Canopy_Height_StdDev_RH100_201904-202004

Replaced with Version 2 layers that span 2019 APR - 2021 AUG:
GEDI_ISS_L3_Elevation_Mean_Lowest_Mode_201904-202108
GEDI_ISS_L3_Elevation_StdDev_Lowest_Mode_201904-202108
GEDI_ISS_L3_Laser_Footprint_Count_201904-202108
GEDI_ISS_L3_Canopy_Height_Mean_RH100_201904-202108
GEDI_ISS_L3_Canopy_Height_StdDev_RH100_201904-202108

@nasa-gibs/worldview

### CODE REVIEW

**Comment by minniewong** [comment]
Awaiting WVS successful build to include snapshots and preview images

File: config/default/common/config/metadata/layers/gedi/GEDI_ISS_L3_Canopy_Height_StdDev_RH100_201904-202108.md [hunk 1]
```diff
@@ -0,0 +1,3 @@
+The Canopy Height Standard Deviation (L3, v2, 2019 APR to 2021 AUG) layer is the standard deviation of RH100 for all valid footprints per 1 km x 1 km grid cell. RH100 or relative height is the 100th percentile of waveform energy relative to ground elevation. This layer is from the version 2 release of the Global Elevation Dynamics Investigation (GEDI) L3 gridded land surface metrics data covering the period between the 19th and the 138th mission weeks (2019-04-19 to 2021-08-04). It is derived from the GEDI L2B version 2 geolocated laser footprints metric data product. The imagery resolution is 1 km and is available globally within -52 and 52 degrees latitude.
```
Comment (jasontk19) [inline config/default/common/config/metadata/layers/gedi/GEDI_ISS_L3_Canopy_Height_StdDev_RH100_201904-202108.md:1]: Should these dates: (2019-04-19 to 2021-08-04) be in the same format as other dates throughout the app? e.g. 2019 APR 19 to 2021 AUG 04?

Also... should the be using a monospace font?

**Review by jasontk19 — CHANGES_REQUESTED** [review]

File: config/default/common/config/metadata/layers/gedi/GEDI_ISS_L3_Canopy_Height_StdDev_RH100_201904-202108.md (hunk 1 above)
Comment (jasontk19) [reply config/default/common/config/metadata/layers/gedi/GEDI_ISS_L3_Canopy_Height_StdDev_RH100_201904-202108.md:1]: Ok but also the description here has two different date formats: "2019 APR to 2021 AUG" and "2019-04-19 to 2021-08-04", is that intentional?

Comment (minniewong) [reply config/default/common/config/metadata/layers/gedi/GEDI_ISS_L3_Canopy_Height_StdDev_RH100_201904-202108.md:1]: yes, unfortunately :(

Comment (minniewong) [reply config/default/common/config/metadata/layers/gedi/GEDI_ISS_L3_Canopy_Height_StdDev_RH100_201904-202108.md]: Should all be in the correct date format now!

**Review by jasontk19 — APPROVED** [review]

File: config/default/common/config/metadata/layers/gedi/GEDI_ISS_L3_Canopy_Height_StdDev_RH100_201904-202108.md (hunk 1 above)
Comment (jasontk19) [reply config/default/common/config/metadata/layers/gedi/GEDI_ISS_L3_Canopy_Height_StdDev_RH100_201904-202108.md]: Looks great!

**Pull request merged**